string object_id
float32 x_offset
float32 y_offset
float32 z_offset
float32 roll
float32 pitch
float32 yaw
bool release
